src/Symfony/Component/Cache/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+7.1
+---
+
+ * Add url decoding of password in `RedisTrait` DSN
```

Comment:
not needed for bug fixes

Reply:
Oh alright, I thought it was considered as a feature. I'll update this as well as the base branch